SPECS/fmt/fmt.spec

```diff
@@ -7,7 +7,8 @@ License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/fmtlib/%{name}
-Source0:        %{url}/archive/%{version}.tar.gz
+#Source0:        %{url}/archive/%{version}.tar.gz
```

Comment:
Nit: please use `%%` in comments. It's not a big deal here except for a warning from `rpm-build` but if the macro is a multi-line one we end up only commenting out its first line and leaving the rest. It's an annoying quirk of the build tool, I think.